package.json

```diff
@@ -4,20 +4,21 @@
     "license": "MIT",
     "scripts": {
         "ng": "ng",
+        "ng-prefer-ipv4": "node --dns-result-order ipv4first ./node_modules/@angular/cli/bin/ng",
         "build": "npm run build-prod:core && npm run build-apps",
         "build:core": "ng build --configuration development core",
         "build-watch:core": "ng build --configuration development core --watch",
         "build-prod:core": "ng build --configuration production core",
         "build-apps": "npm run build-prod:gis && npm run build-prod:dashboards/data-atlas && npm run build-prod:dashboards/ebv-analyzer && npm run build-prod:dashboards/gfbio && npm run build-prod:dashboards/nfdi-gdo",
-        "serve:dashboards/data-atlas:local": "ng serve --configuration development data-atlas --proxy-config proxy-local.conf.json",
+        "serve:dashboards/data-atlas:local": "ng-prefer-ipv4 serve --configuration development data-atlas --proxy-config proxy-local.conf.json",
         "serve:dashboards/data-atlas:nightly": "ng serve --configuration development data-atlas --proxy-config proxy-data-atlas.conf.json",
-        "serve:dashboards/ebv-analyzer:local": "ng serve --configuration development ebv-analyzer --proxy-config proxy-local.conf.json",
+        "serve:dashboards/ebv-analyzer:local": "ng-prefer-ipv4 serve --configuration development ebv-analyzer --proxy-config proxy-local.conf.json",
         "serve:dashboards/ebv-analyzer:nightly": "ng serve --configuration development ebv-analyzer --proxy-config proxy-ebv-analyzer.conf.json",
-        "serve:dashboards/gfbio:local": "ng serve --configuration development gfbio --proxy-config proxy-local.conf.json",
+        "serve:dashboards/gfbio:local": "ng-prefer-ipv4 serve --configuration development gfbio --proxy-config proxy-local.conf.json",
         "serve:dashboards/gfbio:nightly": "ng serve --configuration development gfbio --proxy-config proxy-gfbio.conf.json",
-        "serve:dashboards/nfdi-gdo:local": "ng serve --configuration development nfdi-gdo --proxy-config proxy-local.conf.json",
+        "serve:dashboards/nfdi-gdo:local": "ng-prefer-ipv4 serve --configuration development nfdi-gdo --proxy-config proxy-local.conf.json",
         "serve:dashboards/nfdi-gdo:nightly": "ng serve --configuration development nfdi-gdo --proxy-config proxy-nfdi-gdo.conf.json",
-        "serve:gis:local": "ng serve --configuration development gis --proxy-config proxy-local.conf.json",
+        "serve:gis:local": "ng-prefer-ipv4 serve --configuration development gis --proxy-config proxy-local.conf.json",
```

Comment:
I thought this should work, but it somehow does not:

```
 *  Task wird ausgeführt: npm run serve:gis:local 


> geoengine@0.7.0 serve:gis:local
> ng-prefer-ipv4 serve --configuration development gis --proxy-config proxy-local.conf.json

sh: 1: ng-prefer-ipv4: not found

 *  Der Terminalprozess "/usr/bin/bash '-c', 'npm run serve:gis:local'" konnte nicht gestartet werden (Exitcode: 127). 
 *  Das Terminal wird von Aufgaben wiederverwendet, drücken Sie zum Schließen eine beliebige Taste. 
```

```
node --version
v16.18.1
``

Reply:
Fixed in https://github.com/geo-engine/geoengine-ui/pull/511/commits/7e93b23de08d584ee0e76749bad2acacc823ed8e.
Not sure though whether it's still worth it for the only slightly shorter line :D
I could additionally pull the "serve --proxy-config proxy-local.conf.json" part into the "ng-prefer-ipv4" script, though it becomes kind of hard to comprehend which options are set where then. At least now it's seaparated between node options and angular options.